client/index.js

```diff
@@ -68,7 +83,7 @@ var newConnection = function() {
 	}));
 
 	sock.onclose = function() {
-		console.error("[WDS] Disconnected!");
+		log("error", "[WDS] Disconnected!");
```

Comment:
So you mean instead of using the existing `quiet` option to determine the log level, introduce a separate option?